src/surfaces/drawer/drawer.d.ts

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import DrawerActionBar from './drawerActionBar';
+import DrawerContainer from './drawerContainer';
+import DrawerContent from './drawerContent';
+import DrawerDataCards from './drawerDataCards';
+import DrawerDataGrid from './drawerDataGrid';
+import DrawerDataGroups from './drawerDataGroups';
+import DrawerDeprecatedFiltersDrawer from '../drawerDeprecated/drawerDeprecatedFiltersDrawer';
+import DrawerDetailsWindow from './drawerDetailsWindow';
+import DrawerFiltersDrawer from './drawerFiltersDrawer';
+import DrawerFiltersRail from './drawerFiltersRail';
+import DrawerNavigation from './drawerNavigation';
+import DrawerTitleBar from './drawerTitleBar';
+import DrawerWing from './drawerWing';
+
+export interface DrawerPropTypes {
+    /**
+     * The content of the Drawer.
+     */
+    children?: React.ReactNode;
+    /**
+     * Override or extend the styles applied to the Drawer.
+     */
+    classes?: {
+        root: string;
+    };
+    /**
+     * Add additional classes to the Drawer.
+     */
+    className?: string;
+    /**
+     * Used for DOM testing. https://testing-library.com/docs/queries/bytestid/
+     */
+    dataTestId?: string;
+    dimmer?: boolean;
+    isOpen: boolean;
+    isModal?: boolean;
+    maxHeight?: number;
+    maxWidth?: number | string;
+    onClickOutside?: Function;
+    onClose?: Function;
+    onCloseComplete?: Function;
+    onOpenComplete?: Function;
+    positionX?: 'left' | 'right';
+    positionY?: 'bottom' | 'top';
+    positionYOffset?: number;
+    shadowSize?: 'large' | 'small' | 'xsmall';
+    /**
+     * Supply any inline styles to the Drawer.
+     */
+    style?: {};
```

Comment:
NAB: We should consider `React.CSSProperties` in lieu of `{}` (any object) here.

See https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/basic_type_example/#useful-react-prop-type-examples

```JavaScript
/**
 * Supply any inline styles to the Drawer.
 */
style?: React.CSSProperties;
```

Similar for all `style`  declarations throughout.

Reply:
Changed to object.

Reply:
Looking good.  Resolving.